src/models/utils/index.ts

```diff
@@ -1,3 +1,8 @@
+import { encode } from "ripple-binary-codec/dist"
```

Comment:
You don't need the `/dist` in `src` - that's a testing-specific thing. You treat it like any other package here.